stdlib/LinearAlgebra/src/deprecated.jl

```diff
@@ -5,3 +5,7 @@ rank(A::AbstractMatrix, tol::Real) = rank(A,rtol=tol)
 nullspace(A::AbstractVector, tol::Real) = nullspace(reshape(A, length(A), 1), rtol= tol)
 nullspace(A::AbstractMatrix, tol::Real) = nullspace(A, rtol=tol)
 pinv(A::AbstractMatrix{T}, tol::Real) where T = pinv(A, rtol=tol)
+
+# To be removed in 2.0
+Base.@deprecate /(x::Number, v::AbstractVector)  x*pinv(v)
+Base.@deprecate (\)(a::AbstractVector, b::AbstractArray) = pinv(a) * b
```

Comment:
```suggestion
@deprecate /(x::Number, v::AbstractVector) x*pinv(v)
@deprecate (\)(a::AbstractVector, b::AbstractArray) pinv(a) * b
```

The macro seems to be exported, and there should be no `=` between old and new method. Perhaps we should add/modify tests for these, following the pattern from the docstring:

```julia
@test (@test_deprecated num2hex(1)) == "0000000000000001"
```